File: 프로그래머스/연습문제/Lv2/JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    if not s[0].isdigit(): answer = s[0].upper()
    else: answer = s[0]

    for i in range(1, len(s)):
        if s[i].isdigit() or s[i]==' ': answer += s[i]
        elif s[i-1] == ' ': answer += s[i].upper()
        else: answer += s[i].lower()

    return answer

File: 프로그래머스/연습문제/Lv2/JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    if(!isdigit(s[0])) s[0] = toupper(s[0]);
    for(int i=1; i<s.length(); i++){
        if(isdigit(s[i]) || s[i]==' ') continue;
        else if(s[i-1] == ' ') s[i] = toupper(s[i]);
        else s[i] = tolower(s[i]);
    }
    
    return s;
}
